### ISSUE TITLE
Runtime error on windows 64 

### ISSUE THREAD

**Issue by GrimmSnark** [rochefort-lab/fissa#24]
Hello,

I have been trying to implement this toolbox on a windows 10 64bit computer. However, the code hangs in the first parallel pool selection of core.py (approx line 277). I have tracked the error down to the below: 

 File "C:\ProgramData\Anaconda2\lib\multiprocessing\forking.py", line 358, in get_command_line
    is not going to be frozen to produce a Windows executable.''')
RuntimeError:
            Attempt to start a new process before the current process
            has finished its bootstrapping phase.

            This probably means that you are on Windows and you have
            forgotten to use the proper idiom in the main module:

                if __name__ == '__main__':
                    freeze_support()
                    ...

A similar question has been posted here:
https://stackoverflow.com/questions/18204782/runtimeerror-on-windows-trying-python-multiprocessing

Where do I place the  " if __name__ == '__main__': " to allow the code to run? 

Thanks in advance for your help.

**Comment by swkeemink** [comment]
Thanks for your interest in the toolbox! 

Generally, if this error pops up  in Python, you just have put your main function calls within the main file you're running FISSA from within a simple if statement:
```
import fissa

if __name__ == '__main__':
         code_to_run_fissa
```

Could you tell us a bit on how you implemented FISSA currently? (i.e., just running the example Jupyter notebook, or your own custom .py file, etc.)

I'll try to reproduce it on my own Windows machine. Can you tell me what version of Anaconda you are running? In the past people had to install a newer version of Anaconda to get the toolbox to work.

**Comment by GrimmSnark** [comment]
Hello,

Thank you for your quick reply. I am currently running anaconda version 5.20 with python 2.73. At the moment I am just testing to see how the code works with a modified version of the basic_usage.py

I have just added if __name__ == '__main__': to line 10 and the code seems to be running now, thanks a lot. 

Eventually I want to see if I can incorporate this code into my already developed matlab pipeline. I am testing out various methods of neuropil extraction and your paper detailing this toolbox seems very promising.

**Comment by swkeemink** [comment]
Great! I'll probably also add a new example file for Windows users, for others running into the same problem.

Also interested to see how well you can implement it in a MATLAB pipeline. One useful command will be https://www.mathworks.com/help/matlab/ref/system.html, and then you could just put in ```python basic_usage.py``` for example as the function input, if your Python is set-up globally that should work.

**Comment by GrimmSnark** [comment]
No problem. Yeah I was thinking of  writing the text of the python run file with the file paths for the particular experiment in matlab and using system to call python to run it. When I get it working I will send you an example matlab script. 

Also I am seeing an error when I try and call calc_deltaf which states the below:

Traceback (most recent call last):
  File "C:\PostDoc Docs\code\python\fissa\examples\basic_usage_v2.py", line 27, in <module>
    experiment.calc_deltaf(freq = 1.3)
  File "C:\ProgramData\Anaconda2\lib\site-packages\fissa\core.py", line 442, in calc_deltaf
    raw_f0 = deltaf.findBaselineF0(raw_conc, freq)
  File "C:\ProgramData\Anaconda2\lib\site-packages\fissa\deltaf.py", line 57, in findBaselineF0
    b = scipy.signal.firwin(nfilt, cutoff=fw_base / nyq_rate, window='hamming')
  File "C:\ProgramData\Anaconda2\lib\site-packages\scipy\signal\fir_filter_design.py", line 394, in firwin
    raise ValueError("Invalid cutoff frequency: frequencies must be "
ValueError: Invalid cutoff frequency: frequencies must be greater than 0 and less than fs/2.

I am guessing my sampling freq (1.3Hz) is too low?

**Comment by swkeemink** [comment]
Sorry for the trouble, but I opened a new issue for your frequency problem, so other users can later more easily find the solution (once we have one).

**Comment by scottclowe** [comment]
Hi @GrimmSnark. Thanks for alerting us to this. I've identified the source of the problem and have issued a temporary fix for you over at https://github.com/rochefort-lab/fissa/issues/25#issuecomment-426322942.

I'll close this issue here (regarding multiprocessing on Windows) and we can follow on in the other thread to continue discussing the delta-f/f0 problem.

**Comment by swkeemink** [comment]
I'll keep this open until I've reproduced this error on Windows and made a Windows example .py file as needed.

**Comment by GrimmSnark** [comment]
Hey, so I have  written a matlab function that will write a python file to run the FISSA toolbox on my data. It seems to work pretty well ( see link). I then call a system command in matlab to run the file  like so:

` system(['python "' saveDataPath 'FISSA\FISSA_run.py"']);`

Although I had to add the anaconda version of python to my global environment variable path in windows. 

Link to file: https://github.com/GrimmSnark/Two_photon_imaging/blob/master/basicFunctions/createFISSARunFile.m

**Comment by scottclowe** [comment]
Closed by #40.
